agent/tool-scripts/pprof

```diff
@@ -105,79 +104,40 @@ tool_cmd_file="$tool_output_dir/$tool.cmd"
 tool_pid_file=$pbench_tmp/$group.$iteration.$tool.pid
 tool_output_file=$tool_output_dir/$tool.txt
 export PPROF_TMPDIR=$tool_output_dir
+declare -a hosts=($(cat $inventory | sed -n '/^\[pprof\]/,/^$/p' | awk 'length {if (NR!=1) {print}}'))
```

Comment:
Would be nice to have a single pipe for this

Reply:
You mean `$(sed -n '/^\[pprof\]/,/^$/p'  $inventory| awk 'length {if (NR!=1) {print}}')` ? Or perhaps modifying the sed pattern and getting rid of the awk script?

Reply:
I'm more of an `awk` guy myself... It can do regex pattern/range matches just like awk, I just didn't look into how to get group match to skip stripping the first and last line of the match (the `[pprof]` header and newline in this case).